src/styles/main.scss

```diff
@@ -1,3 +1,34 @@
+@import 'variables';
+@import 'mixins';
+@import 'blocks/header';
+@import 'blocks/card';
+@import 'blocks/stars';
+
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+  color: $main-accent;
+  font-size: 12px;
 }
+
+.catalog {
+  --column-count: 1;
+
+  padding: 50px 48px;
+  display: grid;
+  grid-template-columns: repeat(var(--column-count), 200px);
+  justify-content: center;
+  gap: 46px 48px;
+
+  @media (min-width: 488px) {
+    --column-count: 2;
+  }
+
+  @media (min-width: 768px) {
+    --column-count: 3;
+  }
+
+  @media (min-width: 1024px) {
+    --column-count: 4;
+  }
+}
```

Comment:
Also, add a blank line at the end of all files where it's missing
![image](https://github.com/mate-academy/layout_catalog/assets/63243136/c806a92f-95ac-4132-912e-e0ffdf49d6e8)